Makefile

```diff
@@ -482,7 +482,6 @@ $(CONFIG_IMG): $(CONF_FILES) | $(INSTALLER)
 	$(QUIET): $@: Succeeded
 
 $(PERSIST_IMG): | $(INSTALLER)
-	$(if $(findstring zfs,$(HV)),echo 'eve<3zfs' > $@)
```

Comment:
Is the use of zfs in the image name documented somewhere? If so we need to clean up that documentation.

Reply:
I cannot find any mentions for this kind of check for bytes from persist partition, only in [storage-init](https://github.com/lf-edge/eve/blob/fd0b7a3acc483b069d56a821bec762cd395a1ad9/pkg/storage-init/storage-init.sh#L220).

Reply:
Thanks for checking